projects/demo/src/modules/components/editor/resizable-images/examples/1/index.ts

```diff
@@ -25,6 +25,12 @@ import {switchMap, takeUntil} from 'rxjs/operators';
                 import('@taiga-ui/addon-editor/extensions/starter-kit').then(
                     ({StarterKit}) => StarterKit,
                 ),
+                import('@tiptap/extension-focus').then(({FocusClasses}) =>
```

Comment:
Why do we need it? CSS :focus-within is not enough?

Reply:
I try, it doesn't work :(

Reply:
I think it's doesn't work in `contenteditable`